### ISSUE TITLE
fcitx5 20200211: FTBFS on testing-proposed, amd64

### ISSUE THREAD

**Issue by eatradish** [AOSC-Dev/aosc-os-abbs#2098]
https://github.com/fcitx/fcitx5/commit/7eb183f5d353a2e133284dbbc79e0c673f7c1e57

```
/var/cache/acbs/build/acbs.e6bht85t/fcitx5/src/im/keyboard/keyboard.cpp:23:10: fatal error: emoji_public.h: No such file or directory
   23 | #include "emoji_public.h"
      |          ^~~~~~~~~~~~~~~~
```

**Comment by eatradish** [comment]
fcitx5 newest commit needs `cldr-emoji-annotation`

**Comment by eatradish** [comment]
Fixed with fa01dfe (https://github.com/AOSC-Dev/aosc-os-abbs/commit/fa01dfe6d68976bc80c61cb27799e4cd5a8e7f69), closing.
